.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+
+<!--
+Thank you for your interest in contributing to XCSoar! Please fill out the
+following questions to make it easier for us to review your changes.
+
+You do not need to check all the boxes below all at once, feel free to take
+your time and add more commits. If you're done and ready for review, please
+check the last box.
+-->
+
+
+What is the purpose of this pull request? What does it change?
+--------------------------------------------------------------
+
+<!--
+Please describe the changes here, as detailed as needed.
+-->
+
+
+Has the changes been discussed in an issue or the forum?
+--------------------------------------------------------
+
+<!--
+Please link issues and relevant forum posts here.
+
+If this PR resolves an issue on GitHub, write "Closes #1234" so that the issue
+is closed automatically when this PR is merged.
+-->
+
+
+Checklist
+---------
+
+- [ ] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
```

Comment:
What use is that? With git, I can edit anything I want, and I don't need any GitHub features for that.

Reply:
It's actually really useful. If the contributor did *not* check that maintainer edit checkbox, the XCSoar developers/maintainers cannot make changes to the PR's commits. If it *is* checked, they can, e.g. rebase/squash/fix whatever is needed. I'm not suggesting maintainers should do the work for the contributor, but in some cases it's a lot more effective and less work for the maintainer to just make a change/fix or perhaps squash some commits before merging them. This simply allows for more maintainer control and straight forward workflow for them. I'd recommend we keep it (it's mostly there as a reminder so people don't uncheck it), but if noone else wants it I can remove it.

Reply:
> If the contributor did _not_ check that maintainer edit checkbox, the XCSoar developers/maintainers cannot make changes to the PR's commits.

Not true. I wrote it already: with git, I can edit anything. I don't need GitHub's assistance or allowance.

Reply:
If that was the case, why did GitHub add this feature in the first place?

Reply:
I removed it (and the entire checklist, in fact), and instead made a note about it in the comment at the top. We can remove it if it's important to you or anyone else to not have it, but let's look at the PR template as a whole first now that it's changed a bit.